### DISCUSSION TITLE
视频播放不正常

### DISCUSSION THREAD

**Discussion by bitshine** [AlistGo/alist#244] [Q&A] [answered]
除安卓浏览器外，iphone,mac,win10浏览器及微信均无法正常播放。
播放器连接不上或有声无像。
格式均是mp4，但有一个文件却可以正常播放。

**Comment by xhofe** [comment, marked as answer]
请提供链接。

**Reply by bitshine** [reply]
http://v.kb-zy.com/

**Reply by bitshine** [reply]
IINA, potplayer, vlc 这几个播放器是怎样用的？

**Reply by xhofe** [reply]
> http://v.kb-zy.com/

具体哪个视频

**Reply by xhofe** [reply]
> IINA, potplayer, vlc 这几个播放器是怎样用的？

安装了就能用

**Reply by bitshine** [reply]
1580378789337360.mp4 在微信可以播放，电脑不行。
1640071998687301.mp4 Mac可以播放，ios不行。
其它文件都只有在安卓浏览器正常，其它浏览器都不行。

**Reply by bitshine** [reply]
这些文件在web版阿里云盘都是可以播放的，应该不是转码的问题。

**Reply by xhofe** [reply]
> 1580378789337360.mp4 在微信可以播放，电脑不行。 1640071998687301.mp4 Mac可以播放，ios不行。 其它文件都只有在安卓浏览器正常，其它浏览器都不行。

阿里web端的在线播放全部是转码之后的，本程序均为源文件播放。
- 1580378789337360.mp4 参考https://github.com/Xhofe/alist/discussions/210#discussioncomment-1795946 大部分浏览器不支持mpeg-4 part2视频，基本只支持mpeg-4 part10的视频，也就是H.264的视频
- 1640071998687301.mp4 参考https://github.com/Xhofe/alist/issues/216#issuecomment-993528780
- 其他视频同1，你的视频都太古老了，基本上都是浏览器不支持的编码，识别的resolution都是0 x 0

**Reply by bitshine** [reply]
明白了，谢谢！
